packages/api-contract/src/MetaRegistry.ts

```diff
@@ -0,0 +1,333 @@
+// Copyright 2017-2019 @polkadot/api-contract authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import * as t from '@polkadot/types/types';
+
+import { assert } from '@polkadot/util';
+import { encodeEnum, encodeStruct, encodeTuple } from '@polkadot/types';
+
+const SPECIAL_TYPES = ['AccountId', 'Balance'];
```

Comment:
Not 100% sure what "special types" mean here? (So need to be more descriptive). However, assuming these are the types that have special display/encoding requirements, there are a couple more - `Address` & `AccountIndex`

(These latter 2 are not strictly in-use atm, but they do fall into exactly the same bucket)